src/runtime/server/middleware/session/exceptions.ts

```diff
@@ -0,0 +1,17 @@
+export class SessionHijackAttempt extends Error {
+  constructor (message = 'session-jacking attempt') {
```

Comment:
Nit: I would say that error message can be more neutral as we expect that ip address of the user might change without malicious intention more often than with it:

```suggestion
export class IpChanged extends Error {
  constructor (message = 'IP address has changed') {
```